### PR TITLE
fix comments in github and comments in class

### PR DESCRIPTION
**Pull request by Cxristian**

### CODE REVIEW

File: src/main/java/pivotaltracker/PageTransporter.java [hunk 1]
```diff
@@ -48,6 +48,24 @@ public static void navigatePage(final String keyUrl) {
...
+    public static void navigatePageById(final String keyUrl, final int id, final String permalink) {
+        String baseUrl = ConfigFileReader.getInstance().getProperties().get("baseUrl");
+        goToURL(baseUrl.concat(keyUrl).concat("/").concat(String.valueOf(id)).concat(permalink));
+    }
+
+//    public static String navigatePageByIdString(final String keyUrl, final int id, final String permalink) {
+//        String baseUrl = ConfigFileReader.getInstance().getProperties().get("baseUrl");
+//        goToURL(baseUrl.concat(keyUrl).concat("/").concat(String.valueOf(id)).concat(permalink));
+//        return baseUrl;
+//    }
```
Comment (LimbertVargas) [inline src/main/java/pivotaltracker/PageTransporter.java]: maybe you can delete this commented code

File: src/main/java/pivotaltracker/ui/pages/account/AccountPlansPage.java [hunk 2]
```diff
@@ -54,4 +58,27 @@ public AccountPlansPage() {
     public AccountMenu getAccountMenu() {
         return accountMenu;
     }
+
+    /**
+     * Get url setting of account.
```
Comment (titor-oopart) [inline src/main/java/pivotaltracker/ui/pages/account/AccountPlansPage.java]: Gets is the correct form to write in third person

File: src/main/java/pivotaltracker/ui/pages/CreateProjectPopup.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    /**
+     * Waits until page object is loaded.
+     */
+    @Override
+    public void waitUntilPageObjectIsLoaded() {
+        wait.until(ExpectedConditions.visibilityOf(accountCmb));
+    }
+
+    /**
+     * account select combo box.
```
Comment (LimbertVargas) [inline src/main/java/pivotaltracker/ui/pages/CreateProjectPopup.java]: please add uppercase in "Account" word

**Review by LimbertVargas — COMMENTED** [review]
it's ok.. good job Cristhian

**Pull request merged**